cadc-access-control-admin/src/main/java/ca/nrc/cadc/ac/admin/AbstractUserCommand.java

```diff
@@ -137,7 +137,7 @@ protected void printUser(final User user)
             this.systemOut.println();
             if (user.personalDetails != null)
             {
-                this.systemOut.println(user.personalDetails.toStringFormatted());
+                // Returns users that are locked (disabled or marked deleted)
```

Comment:
This looks like it must be a mistake too.  Doesn't that println needs to be in there for output to show?

Reply:
Or maybe it was moved elsewhere?

Reply:
I think this was a ball drop - I'd meant to remove the comment.  Weird that I didn't catch it in testing, though...

Reply:
Added back in, can see personal details using --view now.